driver/src/main/scala/core/nodeset.scala

```diff
@@ -667,6 +663,41 @@ final class ChannelFactory private[reactivemongo] (
     pipeline
   }
 
+  private def sslContext = {
+
+    val keyManagers: Array[KeyManager] = Option(System.getProperty("javax.net.ssl.keyStore")).map { path =>
+
+      val password = Option(System.getProperty("javax.net.ssl.keyStorePassword")).getOrElse("")
+
+      val ks = {
+        val res = KeyStore.getInstance("JKS")
+        val fin = new FileInputStream(path)
+        res.load(fin, password.toCharArray)
+        res
+      }
+
+      val algorithm = Option(Security.getProperty("ssl.KeyManagerFactory.algorithm")).getOrElse("SunX509")
```

Comment:
It must be documented that it relies on JKS, so people can at least have a look at its specific documentation